UofS-Pulse-Binfo/lims_rawseq
# Faster LIMS Search

This PR removes the old views search for raw sequencing runs and introduces a much faster version. It also has more search filters available.

Testing: 
 - Check that the results are what you expect and match what you got with the old search.
 - Check that you can search for germplasm and see associated runs.
 - Check that filter criteria work
 - Check that paging works.
